kinesis.go

```diff
@@ -76,7 +77,7 @@ func makeParams(action string) map[string]string {
 // RequestArgs store params for request
 type RequestArgs struct {
 	params  map[string]interface{}
-	Records []PutRecordsRecord
+	Records []RecordToPut
```

Comment:
> I didn't contemplate such a change because in the interest of getting my contribution accepted I tried to keep the scope of my changes as narrow as possible.

Makes total sense -- I'm not an Admin on this repo so that decision is out of my hands. I'm sure @le0pard will have an opinion on it.